openapi/workday.expense/types.bal

```diff
@@ -0,0 +1,300 @@
+    string merchant?;
+    # A preview of the instance
+    string descriptor?;
+    # A link to the instance
+    string href?;
+    # Id of the instance
+    string id?;
+};
+
+public type FacetsModelReference FacetsModelReferenceInner[];
+
+public type MultipleInstanceModelReference record {
+    int total?;
+    InstanceModelReference[] data?;
+};
+
+public type Expenseentryservicerepresentation04ce195ed8e1100006b9abb50a91009b record {
+    Expenseentrystatus7db380b67cfa100007fce08f3c4200f2 expenseEntryStatus?;
+    # Amount for Expense Entry
+    record  { }  amount?;
+    # Memo for the Entry
+    string memo?;
+    Entrytype7955a503742f1000117f1e32913906c4 entryType?;
+    # Date of Expense Entry
+    string date?;
+    Scanstatus04ce195ed8e1100008202fa8279800a0 scanStatus?;
+    # CRF for Entries REST operation to expose Attachment related to Optical Character Recognition
+    Expenseentryimage0359e103f67510000bc3d59813370009[] attachments?;
+    # Expense Merchant Name
+    string merchant?;
+    Expenseitem04ce195ed8e1100008202ffbad0800a4 expenseItem?;
+    # Boolean value indicating whether the Mobile Expense has an OCR image.
+    boolean hasOCRReceipt?;
+    # A link to the instance
+    string href?;
+    # A preview of the instance
+    string descriptor?;
+    # Id of the instance
+    string id?;
+};
+
+public type Submitexpensereportrepresentation104b24418b4010000cbe2b6be1762d9a record {
+    Expensereport057f258f13f11000155ba26caf1700e0 expenseReport?;
+    # Id of the instance
+    string id?;
+    # A preview of the instance
+    string descriptor?;
+};
+
+public type Expenseitem04ce195ed8e1100008202ffbad0800a4 record {
+    *InstanceModelReference;
+};
+
+public type Payee9dcae422ca6e1000129b1e7ba3b50065 record {
+    *InstanceModelReference;
+};
+
+public type Expensereport057f258f13f11000155ba26caf1700e0 record {
+    *InstanceModelReference;
+};
+
+public type Scanstatus0855dbb1e2cf1000096a2875ecca0080 record {
+    *InstanceModelReference;
+};
+
+# collection something or other
+public type InlineResponse2001 record {
+    Expenseentryservicerepresentation04ce195ed8e1100006b9abb50a91009b[] data?;
+    int total?;
+};
+
+public type Expensereportstatus8a2cd1fe55c61000112cf72d4dc900d8 record {
+    *InstanceModelReference;
+};
+
+# collection something or other
+public type InlineResponse2002 record {
+    Expenseitemsservicerepresentation978f4a3fe47110001a3243426e540060[] data?;
+    int total?;
+};
+
+# This object represents the possible facets for this resource
+public type FacetsModelReferenceInner record {
+    # A description of the facet
+    string descriptor?;
+    # The alias used to select the facet
+    string facetParameter?;
+    # the facet values
+    record  { # The number of instances returned by this facet
+        int count?; # wid / id / reference id
+        string id; # A description of the facet
+        string descriptor?; # A link to the instance
+        string href?;} [] values?;
```

Comment:
Is this a valid variable name?

Reply:
It complied without any issue. However, replaced with a new component.